CondFormats/GEMObjects/interface/GEMMapItem.h

```diff
@@ -0,0 +1,45 @@
+#ifndef GEMMapItem_h
```

Comment:
hi @jshlee -please use ifndef <subsystem>_<package>_<file>_h here and elsewhere for include guards

Reply:
hi @davidlange6 - I dont quite follow what you mean by the guards. The headers already all have ifndef.

Reply:
sorry - missed the question (and indeed, my comment makes no sense for some reason). The include guards should include not only the name but the subsystem and package name by convention. Eg, CondFormats_GEMObjects_GEMMapItem_h